hackclub.app.yaml

```
@@ -1,10 +1,14 @@
 ---
 "":
   - type: A
-    value: 167.99.113.134
+    value: 188.40.159.192
+  - type: AAAA
+    value: 2a01:4f8:120:144a::192
 "*":
   - type: A
-    value: 167.99.113.134
+    value: 188.40.159.192
+ - type: AAAA
+    value: 2a01:4f8:120:144a::192
```

```suggestion
  - type: AAAA
    value: 2a01:4f8:120:144a::192
```

This should fix the failing CI